src/constants/archanovaConstants.js

```diff
@@ -45,8 +45,10 @@ export const ARCHANOVA_PPN_PAYMENT_PROCESSED = sdkConstants?.AccountPaymentState
 export const SET_ARCHANOVA_WALLET_DEPLOYMENT_ESTIMATE = 'SET_ARCHANOVA_WALLET_DEPLOYMENT_ESTIMATE';
 export const SET_GETTING_ARCHANOVA_WALLET_DEPLOYMENT_ESTIMATE = 'SET_GETTING_ARCHANOVA_WALLET_DEPLOYMENT_ESTIMATE';
 export const SET_CHECKING_ARCHANOVA_SESSION = 'SET_CHECKING_ARCHANOVA_SESSION';
+export const ARCHANOVA_ENS_TRANSFER_METHOD_HASH = 'c7e7cea1'; // transferENSNode(address,address,bytes32,bytes)
+export const ARCHANOVA_WALLET_ENS_MIGRATION = 'ARCHANOVA_WALLET_ENS_MIGRATION';
 
-// transaction tag constants: leave values with SMART_WALLET
+// transaction tag constants: leave VALUES with SMART_WALLET
```

Comment:
Is that intentional?

Reply:
Yes, I previously myself replaced `SMART_WALLET` and values by mistake, just wanted it to be more highlighted.